src/styles/blocks/_product.scss

```
@@ -0,0 +1,86 @@
+.product {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $color-main;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-top: 40px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__code {
+    margin-top: 5px;
+
+    font-size: 10px;
+    color: $color-secondary;
+  }
+
+  &__rating-container,
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rating-container {
+    margin-top: 18px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $color-main-accent;
+  }
+
+  &__price-container {
+    margin-top: 23px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    color: $color-main-accent;
+  }
+
+  &__buy-btn {
+    display: block;
+    height: 40px;
+    border-radius: 5px;
+    text-align: center;
+    border: 1px solid transparent;
+    margin-top: 16px;
+
+    font-size: 14px;
+    line-height: 40px;
+    font-weight: 700;
+    text-transform: uppercase;
+    background-color: $color-blue-accent;
+    color: $color-main;
+
+    transition-duration: 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```